src/EasyAbp.BookingService.Application/EasyAbp/BookingService/Assets/AssetAppService.cs

```diff
@@ -59,7 +59,8 @@ protected override async Task<Asset> MapToEntityAsync(CreateUpdateAssetDto creat
             createInput.Volume,
             createInput.Priority,
             ObjectMapper.Map<TimeInAdvanceDto, TimeInAdvance>(createInput.TimeInAdvance),
-            createInput.Disabled);
+            createInput.Disabled,
+            createInput.ExtraProperties);
     }
```

Comment:
Should use `createInput.MapExtraPropertiesTo(asset)` after the entity is created.